packages/office-ui-fabric-react/src/components/pickers/PeoplePicker/PeoplePickerItems/PickerItemsDefault.scss

```diff
@@ -28,5 +28,10 @@
     vertical-align: middle;
   }
 
-
+  .ms-Persona.ms-Persona--xs {
```

Comment:
This is an override specific to the People Picker? Maybe add a short comment to explain why it's necessary.